tailwind.config.js

```diff
@@ -15,6 +15,16 @@ module.exports = {
     "bg-accent2",
     "bg-accent3",
     "bg-accent4",
+    "bg-accent1/5",
```

Comment:
what is this for again?

Reply:
you can't use a tailwind class dynamically (passing as a prop or whatever) unless you use it somewhere else statically, because it won't be included in the css bundle otherwise
this ensures that the class is always included in the css bundle (it's kind of annoying but it is what it is)